src/styles/optionAlternateTextCompact/base.js

```diff
@@ -0,0 +1,7 @@
+const optionAlternateTextCompact = {
```

Comment:
Rather than using `base.js` and having an extra file, I think it would be OK to just move this to `index.js` and then have the iOS file import the styles from `index.js`.

I was looking through the code, and we don't really have a very well-established pattern for using `base` like this.

Reply:
@tgolen I was having some weird behaviours when `index.ios.js` imported `index.js`, but will check it again

Reply:
@tgolen addressed following @parasharrajat suggestions